abs.yaml

```diff
@@ -0,0 +1,2 @@
+channels:
+  - c3i_test2
```

Comment:
is there a specific reason why we add here this channel?

Reply:
This way bokeh is uploaded to c3i_test2 by prefect. Which is a kind of staging and will allow publishing dask and all its dependencies at once.
But I will have to remove this once all is uploaded.